specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -58,8 +41,8 @@ model InnerErrorModel extends Record<unknown> {
     innererror?: InnerErrorModel;
 }
 
-@doc("Language of the text records. This is BCP-47 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default.")
```

Comment:
Could you not have a `@doc()` on this? It means repeating the comment everywhere it's used. I wonder if `model Language is string` would work. Try it, though I suspect not since `string` is a primitive type.

Reply:
@doc() does not work on an alias, and neither did model Language is string. I just looked over the file again and I don't see a specific reference to this model in the swagger, so I assume that there is a reference to it in another file where @doc() could be added later.

Reply:
Yes. It should be referenced plenty in other swaggers within that directory. That was the intent, anyway.